packages/loot-core/package.json

```diff
@@ -81,5 +81,52 @@
     "webpack-bundle-analyzer": "^4.10.1",
     "webpack-cli": "^5.1.4",
     "yargs": "^17.6.2"
+  },
+  "exports": {
+    "./client/accounts/*": "./src/client/accounts/*.ts",
+    "./client/app/*": "./src/client/app/*.ts",
+    "./client/budgets/*": "./src/client/budgets/*.ts",
+    "./client/data-hooks/schedules": "./src/client/data-hooks/schedules.tsx",
+    "./client/data-hooks/*": "./src/client/data-hooks/*.ts",
+    "./client/modals/*": "./src/client/modals/*.ts",
+    "./client/notifications/*": "./src/client/notifications/*.ts",
+    "./client/prefs/*": "./src/client/prefs/*.ts",
+    "./client/queries/*": "./src/client/queries/*.ts",
+    "./client/redux": "./src/client/redux/index.ts",
+    "./client/store": "./src/client/store/index.ts",
+    "./client/store/mock": "./src/client/store/mock.ts",
+    "./client/users/*": "./src/client/users/*.ts",
+    "./client/platform": {
+      "node": "./src/client/platform.electron.ts",
+      "default": "./src/client/platform.web.ts"
+    },
+    "./client/queries": "./src/client/queries.ts",
+    "./client/query-helpers": "./src/client/query-helpers.ts",
+    "./client/query-hooks": "./src/client/query-hooks.ts",
+    "./client/reports": "./src/client/reports.ts",
+    "./client/shared-listeners": "./src/client/shared-listeners.ts",
+    "./client/SpreadsheetProvider": "./src/client/SpreadsheetProvider.tsx",
+    "./client/transfer": "./src/client/transfer.ts",
+    "./client/undo": "./src/client/undo.ts",
+    "./mocks": "./src/mocks/index.ts",
+    "./platform/client/fetch": {
+      "node": "./src/platform/client/fetch/index.ts",
+      "default": "./src/platform/client/fetch/index.browser.ts"
+    },
+    "./platform/client/undo": "./src/platform/client/undo/index.ts",
+    "./platform/exceptions": "./src/platform/exceptions/index.ts",
+    "./platform/server/asyncStorage": "./src/platform/server/asyncStorage/index.ts",
+    "./platform/server/connection": "./src/platform/server/connection/index.ts",
+    "./platform/server/fetch": "./src/platform/server/fetch/index.web.ts",
+    "./platform/server/fs": "./src/platform/server/fs/index.web.ts",
+    "./platform/server/indexeddb": "./src/platform/server/indexeddb/index.web.ts",
+    "./platform/server/log": "./src/platform/server/log/index.web.ts",
+    "./platform/server/sqlite": "./src/platform/server/sqlite/index.web.ts",
+    "./server/budget/types/*": "./src/server/budget/types/*.d.ts",
+    "./server/*": "./src/server/*.ts",
+    "./shared/*": "./src/shared/*.ts",
+    "./types/models/user": "./src/types/models/user.ts",
```

Comment:
I think this is already exported by `./types/models`

Reply:
Good point tbh. The `user.ts` file was moved in one of the previous PRs to a different location, so this export is no longer relevant. Removed it: https://github.com/actualbudget/actual/pull/4592/commits/bb89fb0dc093b70c598c0af062b627909c15f90e